lnwire/onion_error_test.go

```diff
@@ -180,7 +180,7 @@ func TestFailUnknownPaymentHashOptionalAmount(t *testing.T) {
 	onionError := FailUnknownPaymentHash{}
 
 	var b bytes.Buffer
-	if err := EncodeFailure(&b, onionError, 0); err != nil {
+	if err := EncodeFailure(&b, &onionError, 0); err != nil {
```

Comment:
nit: take reference when var is declared instead

Reply:
Fixed. Also in the `reflect.DeepEqual` below, the same objects were compared.